spec/index.html

```diff
@@ -297,15 +297,48 @@ <h3>IRIs</h3>
     <h3>Literals</h3>
 
     <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a>
-    are basic values that are not IRIs. Examples of literals include
-    strings such as "La Joconde", dates such as "the 4th of July, 1990"
-    and numbers such as "3.14159".
-    Literals are associated with a <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>
-    enabling such values to be parsed and interpreted correctly.
-    String literals can optionally be associated with a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>.
-    For example, "Léonard de Vinci" could
-    be associated with the "fr" language tag and "李奥纳多·达·文西"
-    with the "zh" language tag.</p>
+    are used to encode values, rather than identifying them by IRIs. Examples
+    of literals include strings such as "La Joconde", dates such as
+    "1990-07-04" (the 4th of July, 1990) and numbers such as "3.14159".
```

Comment:
Good call to use backticks (forgot that ReSpec expands those to `<code>`). I like this; just suggest to skip the inner quotes on the non-string values, and perhaps say "ISO-8601 formatted date"? That'd give us:
```suggestion
    of literals include strings such as `"La Joconde"`, ISO-8601 formatted
    dates such as `1990-07-04` and numbers such as `3.14159`.
```

 (XML Schema datatypes, including date, are mentioned in the final paragraph of this section, but mentioning format here to avoid locale ambiguity. Leading with "to encode values" and then call the value itself encoded got a bit too odd; I figure "formatted" might work...)

Reply:
Works for me.